src/IIIFPresentation/Models/API/Collection/Upsert/UpsertFlatCollection.cs

```diff
@@ -14,7 +16,19 @@ public class UpsertFlatCollection
     
     public string? Tags { get; set; }
     
-    public string? Thumbnail { get; set; }
+    public string? PresentationThumbnail { get; set; }
+    
+    [JsonProperty("thumbnail")]
+    public object? ThumbnailFromRequest
```

Comment:
Was this added as part of this PR? I thought this was added in another one?

Reply:
yeah, it was added in #68, which I (wrongly) assumed would be merged by the time we got to this one, so pulled the branch off that - might need to reverse these changes, or merge #68